### PR TITLE
LPD-30443 | Master [playwright]

### PR DESCRIPTION
**Pull request by interaminense**
https://liferay.atlassian.net/browse/LPD-29608

### CODE REVIEW

**Comment by interaminense** [comment]
ci:test:sf

**Comment by interaminense** [comment]
ci:test:analytics-cloud-acceptance

File: modules/test/playwright/tests/osb-faro-web/individuals.spec.ts [hunk 1]
```diff
@@ -0,0 +1,135 @@
...
+			page,
+			projectID: project.groupId,
+		});
+	});
+
+	await test.step('Go to Individuals Dashboard', async () => {
+		await navigateTo({page, pageName: 'Individuals'});
+	});
+
+	await page.waitForTimeout(3000);
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/individuals.spec.ts:101]: Change to use an intelligent wait as `waitForSelector`

File: modules/test/playwright/tests/osb-faro-web/individuals.spec.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+		// it should get the menuitem from the page due to the element
+		// being rendered outside the card.
+
+		await page.getByRole('menuitem', {name: 'abc email'}).click();
+
+		await card.getByLabel('Breakdown Name').click();
+		await card.getByLabel('Breakdown Name').fill('breakdown by email');
+		await card.getByRole('button', {name: 'Save'}).click();
+	});
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/individuals.spec.ts:117]: Need to create a function for it

**Review by ViniLopes20 — COMMENTED** [review]
Need to delete the `IndividualsAttributeBreakdown#AddNewBreakdownByAttributeAndAssertCorrectResultsAppear`

File: modules/test/playwright/tests/osb-faro-web/individuals.spec.ts [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+	const lastTick = ticks.nth(ticksCount - 1);
+
+	const lastTickValue = await lastTick.textContent();
+
+	expect(card.getByText('vinicius@liferay.com')).toBeVisible();
+	expect(lastTickValue).toEqual('1');
+
+	await test.step('Close breakdown tab', async () => {
+		await page.getByLabel('Close').click();
+	});
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/individuals.spec.ts:134]: Delete the property after this step

File: modules/test/playwright/tests/osb-faro-web/individuals.spec.ts [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+		await card.getByRole('button', {name: 'Save'}).click();
+	});
+
+	await waitForLoading(page);
+
+	const ticks = card.locator(
+		'.recharts-cartesian-axis.recharts-xAxis .recharts-layer.recharts-cartesian-axis-tick'
+	);
+	const ticksCount = await ticks.count();
+	const lastTick = ticks.nth(ticksCount - 1);
```
Comment (ViniLopes20) [inline modules/test/playwright/tests/osb-faro-web/individuals.spec.ts:125]: I don't understand this logic 🤔

**Comment by ViniLopes20** [comment]
resend here: https://github.com/liferay-ac/liferay-portal/pull/1672

**Pull request closed without merging**